src/app/importExport.service.ts

```diff
@@ -0,0 +1,22 @@
+import { Injectable } from '@angular/core';
+
+@Injectable({
+    providedIn: 'root'
+  })

```

Comment:
Comment missing, what is this method responsible for? Also missing a general explanation of the service itself

Reply:
This is an essential part of a service. In components, you have to import "component" from angular/core into the typescript file and write @component ({ selector: ...}) to make it an component. Likewise, in services you have to import "injectable" from angular/core and write the code above to make it a service. This enables the class to be injected into components.

Reply:
Thanks for the explanation, can you make a comment in the service with a short explanation?